Sources/Plot/API/HTMLElements.swift

```diff
@@ -426,6 +426,17 @@ public extension Node where Context == HTML.DescriptionListContext {
     }
 }
 
+public extension Node where Context == HTML.DescriptionListContext {
+    /// The `<div>` element wraps a group, that is part of a term-description group in a description list (`<dl>` element).
+    ///
+    /// This is allowed according to the HTML spec: // https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element
```

Comment:
This is a great sentence to add to the commit message, but I don't think it needs to be a part of the public documentation.

Reply:
I agree. I have changed it.